os_vif/__init__.py

```diff
@@ -26,10 +26,9 @@
 
 
 def initialize(reset=False, **config):
-    """
-    Loads all os_vif plugins and initializes them with a dictionary of
-    configuration options. These configuration options are passed as-is
-    to the individual VIF plugins that are loaded via stevedore.
+    """Loads all os_vif plugins and initializes them with a dictionary of
```

Comment:
Would it be possible to just disable checking for this (extremely silly IMHO) flake8 rule?